lab-adam/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "express-demo",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "DEBUG='vehicle*' node server.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "bluebird": "3.4.6",
+    "body-parser": "1.15.2",
+    "chai": "3.5.0",
+    "debug": "2.2.0",
+    "express": "4.14.0",
+    "http-errors": "1.5.0",
+    "mkdirp": "0.5.1",
+    "morgan": "1.7.0",
+    "node-uuid": "1.4.7",
+    "superagent": "2.3.0"
```

Comment:
superagent and chai aren't technically regular dependencies, but you also have them in dev dependencies, so it doesn't break anything.